src/inlineautoformatediting.js

```diff
@@ -141,74 +141,75 @@ export default class InlineAutoformatEditing {
 		} );
 
 		editor.model.document.on( 'change', () => {
-			const changes = editor.model.document.differ.getChanges();
+			const selection = editor.model.document.selection;
 
-			for ( const entry of changes ) {
-				if ( entry.type != 'insert' || entry.name != '$text' ) {
-					continue;
-				}
+			// Do nothing if selection is not collapsed.
+			if ( !selection.isCollapsed || !selection.focus || !selection.focus.parent ) {
```

Comment:
I don't recall ever checking `.focus` not to say `.focus.parent` in selection checks anywhere in the code base. Are these needed? When do they matter?